script/hassfest/quality_scale_validation/strict_typing.py

```diff
@@ -35,4 +55,8 @@ def validate(
             "Integration does not have strict typing enabled "
             "(is missing from .strict-typing)"
         ]
+    if untyped_requirement := _check_requirements_are_typed(integration):
+        return [
+            f"Requirement '{untyped_requirement}' appears untyped (missing py.typed)"
```

Comment:
```suggestion
            f"Requirement '{untyped_requirement}' does not conform PEP 561 (https://peps.python.org/pep-0561/) and should be typed and have a py.typed file."
```
I'm also not happy about this description, but would it make sense to mention the right PEP?

Reply:
I think actually the link to the strict typing IQS documentation may be sufficient:

```
Integration fritz:
* [ERROR] [QUALITY_SCALE] [strict-typing] Requirements ['xmltodict==0.13.0'] do not conform PEP 561 
* [ERROR] [QUALITY_SCALE] Please check the documentation at https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/strict-typing/
```